installation.rst

```diff
@@ -117,505 +83,276 @@ Install Gauge for your OS
 
 .. code-block:: console
 
-    gauge install html-report --file ZIP_FILE_PATH
-
-Download the latest version of plugin from the ``Releases`` section of the respective repository. See `plugin list <//gauge.org/plugins/>`__ to find the repositories.
-
-Updating Plugins
-^^^^^^^^^^^^^^^^
-
-Run this command to update your plugin.
+    sudo yum install gauge
 
 .. code-block:: console
 
-    gauge update <plugin_name>
-
-This downloads the latest compatible plugin from our plugin repository.
+    sudo dnf install gauge
 
 .. note::
-    Every Gauge plugin when published, has metadata indicating the ``min`` and ``max`` version of Gauge that it is compatible with. This is used when installing plugins on a system running a particular gauge version.
+   If you choose to install plugins that are supported by Gauge, see :ref:`install_plugins`.
 
+Verify paths after Gauge Installation
+----------------------------------------
 
-**Example**
+.. admonition:: Postrequisite
 
-.. code-block:: console
+   After installing Gauge on your system, you must verify the installation location of Gauge and Gauge plugins.
 
-    gauge update java
+.. tab-container:: platforms
 
-To update a plugin to a specific version, use the ``--version`` flag.
+    .. tab:: Windows
 
-.. code-block:: console
+       Ensure the following:
 
-    gauge update java --version 0.7.1
+       - The default installation location of Gauge is ``%ProgramFiles%\gauge``.
+       - ``gauge_install_location\bin`` must be in ``PATH`` to run from the command line.
+       - Gauge plugins are installed at ``%APPDATA%\gauge\plugins`` directory.
+       - ``APPDATA`` directory is located at ``C:\Users\USER_NAME\AppData\Roaming``.
 
-You can also update all the installed plugins by running
 
-.. code-block:: console
+    .. tab:: macOS
 
-    gauge update --all
+       Ensure the following:
 
-Read the :ref:`Installation troubleshooting <troubleshoot_plugin_installation>` for more.
+       - The default installation location of Gauge is ``/usr/local/``.
+       - ``usr/local/bin/`` or ``custom_install_location/bin`` must be in ``PATH``.
+       - Gauge plugins are installed under ``~/.gauge/plugins`` directory.
 
-.. note::
+        
+    .. tab:: Debian/APT
 
-    The html-report and screenshot plugins are installed automatically on the first run.
+       Ensure the following:
 
-.. _plugins-uninstallation:
+       - The default installation location of Gauge is ``/usr/local/``.
+       - ``usr/local/bin/`` or ``custom_install_location/bin`` must be in ``PATH``.
+       - Gauge plugins are installed under ``~/.gauge/plugins`` directory.
 
-Uninstall Plugins
------------------
+    .. tab:: YUM/DNF
 
-Plugins can be uninstalled using the ``uninstall`` subcommand. The command is
+       Ensure the following:
 
-``gauge uninstall <plugin-id>``
+       - The default installation location of Gauge is ``/usr/local/``.
+       - ``usr/local/bin/`` or ``custom_install_location/bin`` must be in ``PATH``.
```

Comment:
should we use `[custom_install_location]/bin``?

Reply:
My understanding of the comment is that we follow the convention of using '[ ]' wherever "custom_install_location" is mentioned. Am I right? Without '[ ]' does the meaning differ? I am trying to understand what happens otherwise.

Reply:
This is to keep in consistent with an earlier section in the document. for e.g.

```
     .. code-block:: console
	
	            curl -SsL https://downloads.gauge.org/stable | sh -s -- --location=[custom path]
```